india_compliance/gst_india/doctype/purchase_reconciliation_tool/purchase_reconciliation_tool.js

```diff
@@ -458,11 +471,11 @@ class PurchaseReconciliationTool {
     }
 
     get_supplier_data() {
-        const data = {};
+        this.supplier_data = {};
```

Comment:
When we override the supplier_data, this gives unexpected results.
Including the error that we received when exporting the data, as it called this function again.

Get should always return and not set the data.